drivers/hud_data_quality_report/app/models/hud_data_quality_report/generators/fy2020/question_five.rb

```diff
@@ -0,0 +1,236 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module HudDataQualityReport::Generators::Fy2020
+  class QuestionFive < Base
+    include ArelHelper
+
+    QUESTION_NUMBER = 'Question 5'.freeze
+    QUESTION_TABLE_NUMBER = 'Q5'.freeze
```

Comment:
So these question numbers don't match up with the APR (I think that makes sense) but they are equivalent to those in the APR right?

Reply:
Correct -- the APR specifies the tables in Q5 and Q6 come from the DQ report.